libs/product/driver/in-memory/src/backend/product.service.ts

```diff
@@ -29,41 +29,41 @@ export class DaffInMemoryBackendProductService implements InMemoryDbService {
-      this.productFactory.create({ id: '1007', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1008', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1009', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1010', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1011', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1012', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1013', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1014', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1015', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1016', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1017', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1018', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1019', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1020', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1021', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1022', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1023', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1024', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1025', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1026', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1027', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1028', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1029', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1030', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1031', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1032', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1033', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1034', images: this.productImageFactory.createMany(5) }),
-      this.productFactory.create({ id: '1035', images: this.productImageFactory.createMany(5) }),
+      this.productFactory.create({ id: '1001', url_key: '1001', images: this.productImageFactory.createMany(5) }),
```

Comment:
should this be `url` instead of `url_key`?

Reply:
yes absolutely, fixed